build/tasks/styles.js

```diff
@@ -212,3 +218,35 @@ function createPostCSSProcessor(pluginsListOrMap, options)
 
     return postcss(plugins);
 }
+
+/**
+ * Purge unused styles of minified CSS files.
+ *
+ * @async
+ *
+ * @param  {string} outfile  - The path of the CSS file.
+ * @param  {string} filestem - The CSS file name.
+ */
+async function purgeUnusedCSS(outfile, filestem) {
+
+    const timeLabel = `${filestem}.css purged in`;
+        console.time(timeLabel);
+
+    const purgeCSSContentFiles = Array.from(loconfig.tasks.purgeCSS.content);
+
+    const purgeCSSResults = await new PurgeCSS().purge({
+        content: purgeCSSContentFiles,
+        css: [outfile],
+        rejected: true,
+        defaultExtractor: content => content.match(/[a-z0-9_\-\\\/\@]+/gi) || [],
+        safelist: {
+            standard: [/^((?!\bu-gc-).)*$/]
+        }
+    })
```

Comment:
I recommend whitespace between square brackets of array declarations. It improves readability and distinguishes them from property accessors using bracket notation.

```suggestion
    const purgeCSSResults = await new PurgeCSS().purge({
        content: purgeCSSContentFiles,
        css: [ outfile ],
        rejected: true,
        defaultExtractor: content => content.match(/[a-z0-9_\-\\\/\@]+/gi) || [],
        safelist: {
            standard: [ /^((?!\bu-gc-).)*$/ ]
        }
    })
```